core/css.ts

```diff
@@ -257,6 +257,25 @@ let content = `
   pointer-events: none;
 }
 
+.blocklyFieldCursor {
+  cursor: pointer;
+}
+.blocklyFieldTextInputCursor {
+  cursor: text;
+}
+
+.blocklyFieldDropdownCursor {
+  cursor: pointer;
+}
+
+.blocklyFieldCheckboxCursor {
+  cursor: pointer;
+}
+
+.blocklyFieldColourCursor {
+  cursor: pointer;
+}
+  
```

Comment:
I don't think we actually want to add new CSS classes just for styling the cursors. I think we want to wait for issues like https://github.com/google/blockly/issues/8312 to be completed and then add the style rule there.

(Also you didn't actually apply these classes anywhere so I don't think they'd work anyway :P